charts/oes/values.yaml

```diff
@@ -208,7 +234,7 @@ autopilot:
   image:
     registry: quay.io/opsmxpublic
     repository: ubi8-oes-autopilot
-    tag: v3.7
+    tag: v3.8
```

Comment:
Images should be versioned as v3.8.0, I believe v3.8 is not a correct semantic versioning!

Reply:
handled